### ISSUE TITLE
[DESIGN] - remove black padding from Labs hero and navigation

### ISSUE THREAD

**Issue by noatamir** [Quansight/Quansight-website#244]
### What site is this for?

Quansight Labs

### Expected behavior

Full width menu (grey) & hero (white) without color variations.

### Actual behavior

While testing the new hero illustrations today I realized that the black padding is a design implementation bug. It's consistent across all the pages I can see from storyblok. @irinafumarel helped me define the requirements, and can possibly explain this better 😉. 

<img width="1491" alt="Screenshot 2022-06-21 at 19 43 26" src="https://user-images.githubusercontent.com/6564007/174864720-45e12ced-a1ad-4e84-a35c-213077e5be89.png">

### Anything else?

This might be a bug introduced when we made the heros not black background and the menu not black but grey?

**Comment by tokordys** [comment]
We will need to know the requirements from your end @noatamir. I couldn't assign the issue to Irina unfortunately.

**Comment by noatamir** [comment]
Thanks Irina!
I also mentioned in the issue that we expected no black padding, and that the menu menu (dark grey) & hero (white) should be full width respectively.

**Comment by gabalafou** [comment]
Could @irinafumarel create mockups for this for both desktop and mobile? I tried to close this issue this morning but like @tokordys, I found it difficult to understand and implement the expected design just from the description. For example, I have no idea what "without color variations" was supposed to mean. And yes, we could go back and forth asking clarifying questions, but I feel like Figma would be the clearest way to communicate the expected look for desktop and mobile.
